addons/binding/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/dataexchange/NukiHttpServer.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nuki.dataexchange;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.openhab.binding.nuki.NukiBindingConstants;
+import org.openhab.binding.nuki.dto.BridgeApiLockStateRequestDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+
+/**
+ * The {@link NukiHttpServer} class is responsible for handling the callbacks from the Nuki Bridge.
+ *
+ * @author Markus Katter - Initial contribution
+ */
+public class NukiHttpServer extends AbstractHandler {
```

Comment:
I was looking for a way to register a REST-Endpoint but I did not find any info regarding this.
Another binding spins up (if I remember correctly, actually a bunch) of Jetty servers - that's why I did it this way.
Anyway, any hint/tip on an alternative solution is really appreciated.